Resources/en.lproj/Localizable.strings

```diff
@@ -320,3 +319,10 @@
 
 //EDIT
 "ADD_TO_PLAYLIST" = "Add to Playlist";
+
+"PLAYLIST_PLACEHOLDER" = "Playlist title";
+"PLAYLIST_DESCRIPTION" = "Choose a title for your new Playlist";
+
+"RENAME_PLACEHOLDER" = "New Title";
```

Comment:
Not sure about the capitalization of the word playlist and title here.

Perhaps we should consider choosing either one.

Reply:
you can change it if you like, I don't care I just saw that these were not localized so I fixed it.